### PR TITLE
Add mjs support and refresh mime database

### PR DESCRIPTION
**Pull request by avsm**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+v1.2.0 2021-07-13
+-----------------
+
+* Sync MIME types with latest from Apache (@avsm)
```
Comment (avsm) [inline CHANGES.md]:
```suggestion
* Sync MIME types with latest from Apache (#22 @avsm)
```

**Comment by glondu** [comment]
Is there something in place to prevent mjs being removed at next sync with Apache?

**Comment by avsm** [comment]
No. I can't find mjs in any known mime types db, so I've just added it as you asked for it. There aren't many updates to this database anymore, so I don't think it's a problem in practise.

**Comment by glondu** [comment]
Here is the same request in apache httpd server: https://bz.apache.org/bugzilla/show_bug.cgi?id=61383 (for reference)

**Comment by glondu** [comment]
For the record, mjs appears in https://salsa.debian.org/debian/media-types/-/blob/master/mime.types

**Comment by avsm** [comment]
Thanks, that’s very useful — we can use the Debian mime.types for the next update.

**Pull request merged**